server/README.md

```diff
@@ -0,0 +1,39 @@
+# Dose Amigos Server Side Code
+
+## Build instructions
+Run `./gradlew clean build` to build and run tests.
+`clean` will delete the build folder and `build` will create a build folder.
+In `build/distributions`, a file called `server.zip` will be created that contains all dependency jars as well as compiled classes.
+This is what is added to the lambdas
+
+## Code Conventions
+All classes will be contained in a package associated with the feature it belongs to.
+For example `Med.java` would belong in the `info.doseamigos.med` package.
+
+The code will be written in Java 8, so that it's compatible with Amazon AWS Lambdas.
+
+We'll handle dependencies between classes using Google Guice: https://github.com/google/guice
```

Comment:
I looked up other examples of people using Spring with lambdas, and it looks like a giant hassle either way.  I find guide to be more lightweight and easier to understand myself.  What sucks is that we'll probably have to create injectors in each handler that we specify to lambda.   Not the end of the world, just not great.